packages/contracts/src/RelayHub.sol

```diff
@@ -31,6 +31,7 @@ contract RelayHub is IRelayHub, Ownable {
 
     IStakeManager public immutable override stakeManager;
     address public immutable override penalizer;
+    address public override batchGateway;
```

Comment:
should be immutable. just like all other code-related items (sm, penalizer)

Reply:
Fixed.

Reply:
no, it isn't.
its: `address public override batchGateway`
which is not immutable...

Reply:
Now fixed.